.dockerignore

```diff
@@ -0,0 +1 @@
+.node_modules
```

Comment:
I think this should be just `node_modules` without the `.` or `**/node_modules` which I believe should match that folder name in any directory according to the [Docker docs](https://docs.docker.com/engine/reference/builder/)